python/hail/expr/aggregators/aggregators.py

```diff
@@ -1015,7 +1015,28 @@ def hist(expr, start, end, bins) -> StructExpression:
                 bin_freq=tarray(tint64),
                 n_smaller=tint64,
                 n_larger=tint64)
-    return _agg_func('Histogram', expr, t, [start, end, bins])
+    return _agg_func('Histogram', expr, t, constructor_args=[start, end, bins])
+
+
+@typecheck(x=expr_float64, y=expr_float64, n_divisions=int)
+def downsample(x, y, n_divisions=500) -> ArrayExpression:
+    """Downsample (x, y) coordinate datapoints.
+
+    Parameters
+    ---------
+    x : :class:`.NumericExpression`
+        X-values to be downsampled.
+    y : :class:`.NumericExpression`
+        Y-Values to be downsampled.
+    n_divisions : :obj:`int`
+        Factor by which to downsample (default value = 500). A lower input results in fewer output datapoints.
+
+    Returns
+    -------
+    :class:`.ArrayExpression`
+        Expression for downsampled coordinate points (x, y).
```

Comment:
Can you add that the element type of the Array is :py:data:\`.ttuple\` of  :py:data:\`.tfloat64\` and   :py:data:\`.tfloat64\`?